uint8 penalty
uint8 secs_till_unpenalised
